### PR TITLE
Align switch with top of label and ensure that the label wraps

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
blackbaud/skyux-forms#19
blackbaud/skyux-forms#20

Related `list-builder-view-checklist` issue blackbaud/skyux-list-builder-view-checklist#21

### CODE REVIEW

File: src/app/public/styles/_switch.scss [hunk 1]
```diff
@@ -60,6 +59,7 @@ $sky-switch-size: 22px !default;
   height: $sky-switch-size;
   flex: 1 0 $sky-switch-size;
   margin: auto;
+  margin-top: 0;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/styles/_switch.scss]: We've got some overlapping margin styles on these few lines. How about we roll them together in some shorthand: 
`margin: 0 5px auto auto;`

File: src/app/public/styles/_switch.scss [hunk 2]
```diff
@@ -3,7 +3,6 @@ $sky-switch-size: 22px !default;
 .sky-switch {
   cursor: pointer;
   display: inline-flex;
-  white-space: nowrap;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/styles/_switch.scss]: Wouldn't this cause any checkbox labels that aren't in a `sky-list-view-checklist` to now wrap?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/styles/_switch.scss]: Correct. Which is what they are supposed to be doing https://github.com/blackbaud/skyux-forms/issues/19

File: src/app/public/styles/_switch.scss (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/styles/_switch.scss]: Good call. Done

**Comment by Blackbaud-SteveBrush** [comment]
@Blackbaud-TrevorBurch Think we could create a visual test that demonstrates the vertical alignment of the checkbox/radio, as well as the wrapping text?

File: src/app/public/styles/_switch.scss (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/styles/_switch.scss]: 👍

**Pull request merged**